app/api/email_utils.py

```diff
@@ -137,6 +137,48 @@ def send_email_mentorship_relation_accepted(request_id):
     )
     send_email(request_sender.email, subject, html)
 
+def send_email_report_violation(user_id, comment_id):
+    """
+    Sends an email to the admins as a notification,
+    for a violation reported by a user on a task comment.
+
+    Args:
+        user_id: ID of user reporting the violation
+        comment_id: ID of comment that is being reported
+    """
+
+    from app.database.models.user import UserModel
+    from app.database.models.task_comment import TaskCommentModel
+
+    # User who's reporting the violation
+    user = UserModel.find_by_id(user_id)
+    # Get the name of the user
+    user_name = user.name
+    # Email of user
+    user_email = user.email
+
+    # Get task comment
+    task_comment = TaskCommentModel.find_by_id(comment_id)
+    # Comment string
+    comment = task_comment.comment
+
+    # Commentor of the task comment being reported
+    commenter = UserModel.find_by_id(task_comment.user_id)
+    # Commentor name
+    commenter_name = commenter.name
+    # Commentor email
+    commenter_email = commenter.email
+
+    subject = "Violation Reported"
```

Comment:
Could this have a bit more context? Maybe "Violation Reported on Task Comment #id"

Reply:
Our team had this discussion actually, and we decided to include the email of the user and the commenter but not the ids. For the id of the task comment, I guess two different task comments for two different relations/tasks can have the same id.

Reply:
Makes sense, I was going for the context more, to add Task Comment or the Application name?

Reply:
Done